ext/mysqli/mysqli_api.c

```diff
@@ -126,7 +126,7 @@ int mysqli_stmt_bind_param_do_bind(MY_STMT *stmt, unsigned int num_vars, zval *a
 PHP_FUNCTION(mysqli_stmt_bind_param)
 {
 	zval			*args;
-	int				argc;
+	uint32_t		argc;
```

Comment:
Shouldn't the signature of `mysqli_stmt_bind_param_do_bind` be changed too?

Reply:
I did this, and changed some more signatures so that it returns `enum_func_status`, which I think a lot of other functions in mysqli should actually be returning instead of a random int